packages/cozy-harvest-lib/src/models/ConnectionFlow.spec.js

```diff
@@ -1,4 +1,8 @@
 import EventEmitter from 'events'
+import { waitFor } from '@testing-library/react'
+
+import { Q } from 'cozy-client'
```

Comment:
how previous version of this file can have been worked? :o Are we ignoring somehow the result of the tests?

Reply:
This is rebase error after eslint changed the order of imports. I fix it